src/server/page_server.cpp

```diff
@@ -143,6 +148,17 @@ void page_server::receive_disconnect_request (cubpacking::unpacker &upk)
   disconnect_thread.detach ();
 }
 
+void page_server::receive_boot_info_request (cubpacking::unpacker &upk)
```

Comment:
it would have been better to maintain a certain naming convention for these functions; if the request is `ats_to_ps_request::GET_BOOT_INFO`, then the handler should have something derived from the name of the constant (ie: `on_get_goot_info` or `on_ats_to_ps_request_get_boot_info`);

Reply:
the new functions follow the existing naming convention. maybe suggested naming convention is better, but it must be done in a separate refactoring task, because other functions should be renamed too.